manifests/r/RemoteUtilities/Host/7.1.7.0/RemoteUtilities.Host.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: RemoteUtilities.Host
+PackageVersion: 7.1.7.0
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2012.x64
Installers:
```

Reply:
Research yielded a mapping of DLL name to VCRedist package version: 
MSVCR71.dll: 2005?
MSVCR08.dll: 2008
MSVCR09.dll: 2010
MSVCR120.dl:| 2012
MSVCR130.dll: 2013
MSVCR140.dll: 2015+

Based on this, the Microsoft.VCRedist.2012.x64 is necessary. 
- Testing in my VM provided an odd state: after installing dependency and then package, the executables still error that the DLL is missing. Running the same `winget install` command and running the installer again were necessary to clear the error - only after this 2nd installer run did the executables launch without error. The VCRedist dependency did not reinstall on the 2nd run.